controller/kubernetes_configmap_controller.go

```diff
@@ -267,3 +284,18 @@ func isLonghornStorageClass(obj interface{}) bool {
 
 	return sc.Name == types.DefaultStorageClassName && sc.Provisioner == types.LonghornDriverName
 }
+
+func (kc *KubernetesConfigMapController) UpdateBuiltInSettingsWithCustomizedValues(defaultSettingCM *v1.ConfigMap) error {
```

Comment:
This function is actually the same as `types.UpdateBuiltInSettingsWithCustomizedValues`.

Reply:
Addressed. Thank you